### PR TITLE
Important Formatting for a better overview

### PR DESCRIPTION
**Pull request by Konrad570**
Formatting is very important for an better Overview of the Code
Specially in some Code Editors and Screen sizes the Code is very hard to read and editable. This was a few good format to prevend any failure with spaces etc.

### CODE REVIEW

File: src/command/ig.jl [hunk 1]
```diff
@@ -291,7 +291,7 @@ end
 function ig_reformat_view!(df::AbstractDataFrame)
     select!(df, Not(:purchase_price))
     rename!(df, "current_price" => "price", "market_value" => "amount")
-    return df
+return df
```
Comment (Azzaare) [inline src/command/ig.jl:294]: This change is actually a mistake, `return` should be indented

File: src/command/ig.jl [hunk 2]
```diff
@@ -538,7 +538,7 @@ function ig_sell_fifo(pf::IgPortfolio, symbol::AbstractString, shares::Real, cur
                 remaining = 0
             else # relief this lot completely and continue
                 remaining -= h.shares
-            end
+end
```
Comment (Azzaare) [inline src/command/ig.jl:541]: Here also is very odd

File: src/command/jc.jl [hunk 3]
```diff
@@ -57,7 +57,7 @@ end
 
 function jc_execute(c, m, ::Val{:next}, args)
     @info "Calling :next"
-end
+    end
```
Comment (Azzaare) [inline src/command/jc.jl:60]: Here too

**Review by Azzaare — CHANGES_REQUESTED** [review]

**Comment by Azzaare** [comment]
Hi, thanks for the PR.

Even though we forgot to mention it in the README, we try to follow the following guidelines: https://github.com/invenia/BlueStyle

In your PR, many changes are great for visibility. Some other changes go against the abovementioned guideline. And a few also introduce bad indenting.
It is probably necessary to build upon this PR to make it converge to something adequate.

**Comment by Konrad570** [comment]
Okay I see, what about the other changes that aren’t odd, will you change this in the upcoming updates?

**Comment by Azzaare** [comment]
I have used an automated formatting tool set on Blue Style and made a PR based on your fork.
It is not perfect, but as you can see, some changes you made conflict with that style (and some don't).

As this is a collaborative project, it seems hard to move to another style without approval of most collaborators. If you feel that Blue Style is not satisfying for you, please let us know why and what are the critical parts are.

**Comment by tk3369** [comment]
Let's close this issue since the standard julia formatter from vscode is broken. See #118 for a new attempt.

**Pull request closed without merging**